grid/social-media/posts/maps/__snapshots__/linkedin.test.ts.snap

```diff
@@ -20,169 +20,142 @@ Ok {
       Object {
         "attachments": Array [
           Object {
-            "altText": undefined,
-            "description": undefined,
-            "duration": undefined,
-            "height": undefined,
-            "preview": "http://jrs-stage.int.railwaymen.org/rails/active_storage/blobs/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBa3U2IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--edd7dcfe90f218434c9f632d5767e04a96531f4d/PI-PNL-02-0.jpg",
-            "title": "1 test estates link 1",
-            "type": "link",
-            "url": "http://jrs-stage.int.railwaymen.org/content/8f36d0af-bcc8-4294-8708-bd6baab80424",
```

Comment:
note: Missing URL and preview URL of images in attachments makes the profile much less usable.

Reply:
Yup, this is actually holding me back from making this PR ready for review. I'm exploring how to fix it.